tests/python/test.py

```diff
@@ -14,7 +15,7 @@ def test_underscorize():
         "CamelBI1GCasesRun1": "camel_bi1g_cases_run1"
     }
     for example, expected in camel_to_underscored_expectations.items():
-        assert underscorize(example) == expected
+        asserts.eq_(underscorize(example), expected)
```

Comment:
I thought about that, but I like that is still reads as "assert equal", though—what think you?